src/sssom/parsers.py

```diff
@@ -977,6 +978,27 @@ def split_dataframe(
     )
 
 
+@dataclass(frozen=True, eq=True)
+class SSSOMSplitTriple:
```

Comment:
why not use a named tuple here?

Reply:
Definitely could do! Although that would preclude having methods on the class. (Those methods were for convenience, since they're used twice).

Reply:
Yes, true, a subclass of NamedTuple could add those methods. I am happy to change that- though what is the advantage over a hashable dataclass (i.e. `frozen=True`) at that point?

Reply:
Oh wow, subclassing NamedTuple is a good deal more ergonomic than it used to be. Change incoming!